pandas/tests/indexes/multi/test_get_level_values.py

```diff
@@ -89,3 +89,17 @@ def test_get_level_values_na():
     result = index.get_level_values(0)
     expected = pd.Index([], dtype=object)
     tm.assert_index_equal(result, expected)
+
+
+def test_get_level_values_when_periods():
+    # GH33131. See also discussion in GH32669.
+    # This test can probably be removed when PeriodIndex._engine is removed.
+    from pandas import Period, PeriodIndex
+
+    idx = MultiIndex.from_arrays(
+        [PeriodIndex([Period("2019Q1"), Period("2019Q2")], name="b")]
+    )
+    idx2 = MultiIndex.from_arrays(
+        [idx._get_level_values(level) for level in range(idx.nlevels)]
+    )
+    assert all(x.is_monotonic for x in idx2.levels) is True
```

Comment:
Question not a blocker.

Is the ```is True``` necessary?

Reply:
Yeah, in this case it's not needed. I'm in the habit of testing properties with a strict ``is`` test, where possible. I'll remove it, but will wait to see if other comments need addressing.